playbook.yml

```diff
@@ -11,161 +11,223 @@
-      blockinfile:
-        path: /etc/sysconfig/network
-        block: |
-          RES_OPTIONS="rotate timeout:1 attempts:1"
\ No newline at end of file
+  - name: Make directory structure
+    file:
+      path: /{{ item }}
+      state: directory
+      mode: 0775
+      recurse: yes
+    with_items:
+      - ADT
+      - EF
+      - NATS
+      - appdata
+      - appdata/ADT
+      - appdata/ADT/archive
+      - appdata/ADT/archive/07032016acl
+      - appdata/ADT/archive/07032016oag
+      - appdata/ADT/archive/acl
+      - appdata/ADT/archive/mvt
+      - appdata/ADT/archive/oag
+      - appdata/ADT/data
+      - appdata/ADT/data/07032016acl
+      - appdata/ADT/data/07032016oag
+      - appdata/ADT/data/acl
+      - appdata/ADT/data/mvt
+      - appdata/ADT/data/oag
+      - appdata/ADT/quarantine
+      - appdata/ADT/quarantine/acl
+      - appdata/ADT/quarantine/mvt
+      - appdata/ADT/quarantine/oag
+      - appdata/ADT/scripts
+      - appdata/ADT/stage
+      - appdata/ADT/stage/acl
+      - appdata/ADT/stage/maytech
+      - appdata/ADT/stage/mvt
+      - appdata/ADT/stage/oag
+      - appdata/EF
+      - appdata/EF/data
+      - appdata/EF/data/drt
+      - appdata/EF/keys
+      - appdata/EF/log
+      - appdata/EF/scripts
+      - appdata/NATS
+      - appdata/NATS/archive
+      - appdata/NATS/archive/nats
+      - appdata/NATS/data
+      - appdata/NATS/data/nats
+      - appdata/NATS/data/ga
+      - appdata/NATS/log
+      - appdata/NATS/quarantine
+      - appdata/NATS/quarantine/nats
+      - appdata/NATS/scripts
+      - appdata/NATS/stage
+      - appdata/NATS/stage/maytech
+      - appdata/NATS/stage/nats
+
+  - file:
+      src: /appdata/ADT
+      dest: /ADT
+      force: yes
+      state: link
+
+  - file:
+      src: /appdata/EF
+      dest: /EF
+      force: yes
+      state: link
+
+  - file:
+      src: /appdata/NATS
+      dest: /NATS
+      force: yes
+      state: link
+
+  - name: SSM Group
+    group:
+      name: SSM
+      system: yes
+
+  - name: Create wherescape user
+    user:
+      name: wherescape
+      group: SSM
+
+  - name: Yum Install Dependencies
+    yum: name={{ item }}
+    with_items:
+      - epel-release
+      - python-pip
+      - gcc
+      - pcre-devel
+      - openssl-devel
+      - zlib-devel
+      - wget
+      - git
+      - cloud-utils
+
+  - name: Pip Install Dependencies
+    pip: 'name={{ item }}'
+    with_items:
+      - virtualenv
+      - AWSCLI
+      - ftputil
+
+  - name: Install Python 2.7
+    shell: '{{ item }}'
+    args:
+      chdir: /usr/src
+    with_items:
+      - wget https://www.python.org/ftp/python/2.7.10/Python-2.7.10.tgz
+      - tar xzf Python-2.7.10.tgz
+
+  - name: Configure Python 2.7
+    shell: ./configure
+    args:
+      chdir: /usr/src/Python-2.7.10
+
+  - name: GET config files from DQ-config-bucket
+    shell: '{{ item }}'
+    with_items:
+      - aws configure set aws_access_key_id '{{user_access_key}}'
```

Comment:
what sets `user_access_key`? can't you add an iam profile to the packer instance?
https://www.packer.io/docs/builders/amazon-instance.html#iam_instance_profile

Reply:
set in the packer instance already:
```
{
  "variables": {
    "access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
    "secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
  },
```